MSAL/test/XCTestCase+HelperMethods.m

```diff
@@ -15,18 +17,34 @@
 //
 // THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 // IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
 // AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
 
-#pragma once
-
-#import <Foundation/Foundation.h>
+#import "XCTestCase+HelperMethods.h"
 
-@interface MSALIpAddressHelper : NSObject
+@implementation XCTestCase (HelperMethods)
 
-+ (NSString *)msalDeviceIpAddress;
+- (void)msalAssertStringEquals:(NSString *)actual
```

Comment:
why do we need to add custom handling for this?

Reply:
We need it in order to compare strings in tests by value, not by pointer (we do same in ADAL).

Reply:
I think that's not an issue anymore (used to be some years ago). XCTAssertEqualObjects should be able to do the same. Correct me if I'm wrong. If you agree, we should get rid of msalAssertStringEquals.